go/base/context.go

```diff
@@ -279,49 +297,59 @@ func (this *MigrationContext) GetOldTableName() string {
 func (this *MigrationContext) GetChangelogTableName() string {
 	if this.ForceTmpTableName != "" {
 		return getSafeTableName(this.ForceTmpTableName, "ghc")
-	} else {
-		return getSafeTableName(this.OriginalTableName, "ghc")
 	}
+	return getSafeTableName(this.DstTableName, "ghc")
 }
 
 // GetVoluntaryLockName returns a name of a voluntary lock to be used throughout
 // the swap-tables process.
 func (this *MigrationContext) GetVoluntaryLockName() string {
-	return fmt.Sprintf("%s.%s.lock", this.DatabaseName, this.OriginalTableName)
+	return fmt.Sprintf("%s.%s.lock", this.SrcDatabaseName, this.SrcTableName)
 }
 
 // RequiresBinlogFormatChange is `true` when the original binlog format isn't `ROW`
 func (this *MigrationContext) RequiresBinlogFormatChange() bool {
 	return this.OriginalBinlogFormat != "ROW"
 }
 
-// GetApplierHostname is a safe access method to the applier hostname
-func (this *MigrationContext) GetApplierHostname() string {
-	if this.ApplierConnectionConfig == nil {
+// GetDstHostname is a safe access method to the applier master hostname
+// NOTE: use master for dst to keep it same with previous
+func (this *MigrationContext) GetDstHostname() string {
+	if this.DstMasterConnectionConfig == nil {
 		return ""
 	}
-	if this.ApplierConnectionConfig.ImpliedKey == nil {
+	if this.DstMasterConnectionConfig.ImpliedKey == nil {
 		return ""
 	}
-	return this.ApplierConnectionConfig.ImpliedKey.Hostname
+	return this.DstMasterConnectionConfig.ImpliedKey.Hostname
 }
 
-// GetInspectorHostname is a safe access method to the inspector hostname
-func (this *MigrationContext) GetInspectorHostname() string {
-	if this.InspectorConnectionConfig == nil {
+// GetSrcHostname is a safe access method to the inspector replica hostname
+func (this *MigrationContext) GetSrcHostname() string {
+	if this.SrcConnectionConfig == nil {
 		return ""
 	}
-	if this.InspectorConnectionConfig.ImpliedKey == nil {
+	if this.SrcConnectionConfig.ImpliedKey == nil {
 		return ""
 	}
-	return this.InspectorConnectionConfig.ImpliedKey.Hostname
+	return this.SrcConnectionConfig.ImpliedKey.Hostname
 }
 
-// InspectorIsAlsoApplier is `true` when the both inspector and applier are the
+// SrcIsAlsoDst is `true` when the both source and destination are the
 // same database instance. This would be true when running directly on master or when
 // testing on replica.
-func (this *MigrationContext) InspectorIsAlsoApplier() bool {
-	return this.InspectorConnectionConfig.Equals(this.ApplierConnectionConfig)
+func (this *MigrationContext) SrcIsAlsoDst() bool {
+	return this.DstMasterConnectionConfig.Equals(this.SrcMasterConnectionConfig)
+}
+
+// SrcMasterIsAlsoReplica is `true` when the master and replica is the same on source
+func (this *MigrationContext) SrcMasterIsAlsoReplica() bool {
```

Comment:
name  SrcMasterIsAlsoReplica and DstMasterIsAlsoReplica are very strange，and "the master and replica is the same on source" not make sense。

Reply:
but names "src srcMaster dst dstMaster" it's clear，i dont have  better idea。